### PR TITLE
[hi] Localize i18n strings in hi.toml

### PR DESCRIPTION
**Pull request by 7h3-3mp7y-m4n**
### Describe your changes
changed  the hi.toml file which have English text into proper Hindi with correct grammar 
Issue#2182[https://github.com/cncf/glossary/issues/2182]

### Checklist before opening this PR (put `x` in the checkboxes)
- [x ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by 7h3-3mp7y-m4n** [comment]
I just add my signed off in my commit and I'm facing issue can you tell me why I'm facing this

File: i18n/hi.toml [hunk 1]
```diff
@@ -20,21 +20,21 @@ other = "में"
...
 # Phrases for tags
 [ui_see_all_tags]
-other = "See all tags"
+other = "सभी टैग को देखे"
 [ui_tag]
-other = "Tag"
+other = "टैग""
 [ui_tags]
-other = "Tags"
+other = "टैग""
```
Comment (jihoon-seo) [inline i18n/hi.toml:27]: These redundant quotation marks are causing [Netlify build failure](https://app.netlify.com/sites/cncfglossary/deploys/64a7f372efdf390007f953a6#L74).

```diff
[ui_tag]
-other = "टैग""
+other = "टैग"
[ui_tags]
-other = "टैग""
+other = "टैग"
```

File: i18n/hi.toml [hunk 2]
```diff
@@ -45,7 +45,7 @@ other = "गोपनीयता नीति"
 
 
 [footer_hub_button_text]
-other = "All CNCF Sites"
+other = "सभी CNCF साइटें""
```
Comment (jihoon-seo) [inline i18n/hi.toml:48]:
```suggestion
other = "सभी CNCF साइटें"
```

**Comment by jihoon-seo** [comment]
@7h3-3mp7y-m4n Thank you for your contribution, but
from [this comment](https://github.com/cncf/glossary/pull/2197#issuecomment-1623068659),
> this repo does not allow localization contribution to the main branch directly. you need to use `dev-hi` branch for the localization. Hindi localization approvers are supposed to take care localization PRs.

Please branch off your feature branch from `dev-hi`, and set the target branch of your PR to `dev-hi` as well.

**Comment by 7h3-3mp7y-m4n** [comment]
thx for explaining @jihoon-seo  yes I'm looking forward to do that

**Pull request closed without merging**